lib/metasploit/framework/data_service/proxy/vuln_data_proxy.rb

```diff
@@ -5,7 +5,7 @@ def report_vuln(opts)
       data_service = self.get_data_service()
       data_service.report_vuln(opts)
     rescue  Exception => e
-      elog "Problem reporting vulnerability: #{e.message}"
+      self.log_error(e, "Problem ")
```

Comment:
Did you intend to remove `reporting vulnerability`?

Reply:
Nope, that was an accident.